.github/workflows/pages.yml

```diff
@@ -0,0 +1,61 @@
+# Bundle app with production url and deploy to GitHub Pages
+name: Deploy github pages
+
+on:
+  push:
+    branches: ["master"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+env:
+  NOTIFICATION_URL: https://ddehnr4ewobxc.cloudfront.net/dynNotifications.json
```

Comment:
Can we use the URL from env files here? Just to keep a single source of truth?
Also why are deploying on gh-pages?

Reply:
Removed env from workflow file.